### ISSUE TITLE
Can't customize active step text style in NumberStepper

### ISSUE THREAD

**Issue by Nashati** [imujtaba8488/package_im_stepper#19]
Any idea how I can achieve that?

![image](https://user-images.githubusercontent.com/29517209/108641897-7e782f00-74aa-11eb-8ae8-37bbb19bb591.png)

**Comment by imujtaba8488** [comment]
There is already a numberStyle property. However, are you suggesting a different style than the rest for the active step?

**Comment by Nashati** [comment]
Yeah, I think having a different text style for the active step is a legitimate option. I didn't find a way to change the text style of only the active step.
So I suggest to add `activeNumberStyle`, the same way that you added `activeStepColor`, `activeStepBorderColor` etc...

**Comment by irshad-hz** [comment]
is it possible to add straight lines insted of doted lines?

**Comment by abisao1** [comment]
> Yeah, I think having a different text style for the active step is a legitimate option. I didn't find a way to change the text style of only the active step. So I suggest to add `activeNumberStyle`, the same way that you added `activeStepColor`, `activeStepBorderColor` etc...

You could also add border for inactive step.
